internal/service/project/resource_project.go

```diff
@@ -301,12 +320,13 @@ func (r *projectRS) Create(ctx context.Context, req resource.CreateRequest, resp
 	if resp.Diagnostics.HasError() {
 		return
 	}
-
+	tags := *NewAdminTags(ctx, projectPlan.Tags)
 	projectGroup := &admin.Group{
 		OrgId:                     projectPlan.OrgID.ValueString(),
 		Name:                      projectPlan.Name.ValueString(),
 		WithDefaultAlertsSettings: projectPlan.WithDefaultAlertsSettings.ValueBoolPointer(),
 		RegionUsageRestrictions:   projectPlan.RegionUsageRestrictions.ValueStringPointer(),
+		Tags:                      &tags,
```

Comment:
related to above comment, could we just have:  tags := NewAdminTags(ctx, projectPlan.Tags) if we later take the address?

Reply:
fixed in 35d729e4e186219cab55b274cb1af334b82e1645